stdlib/2and3/colorsys.pyi

```diff
@@ -0,0 +1,17 @@
+# Stubs for colorsys
+
+from typing import Tuple, Union
+
+def yiq_to_rgb(y: float, i: float, q: float) -> Tuple[float, float, float]: ...
+def rgb_to_hls(r: float, g: float, b: float) -> Tuple[float, float, float]: ...
+def hls_to_rgb(h: float, l: float, s: float) -> Tuple[float, float, float]: ...
+def rgb_to_hsv(r: float, g: float, b: float) -> Tuple[float, float, float]: ...
+def hsv_to_rgb(h: float, s: float, v: float) -> Tuple[float, float, float]: ...
+
+# TODO undocumented
+ONE_SIXTH = ...  # type: float
+ONE_THIRD = ...  # type: float
+TWO_THIRD = ...  # type: float
+
+# TODO undocumented, should only work with float
+def rgb_to_yiq(r: complex, g: complex, b: complex) -> Tuple[float, float, float]: ...
```

Comment:
Don't worry, it's what PR are for.
Back to float then (mypy should also help to see if someone use the stdlib in a weird/unsupported way).